src/telemetry/mod.rs

```diff
@@ -70,9 +70,19 @@ pub fn init_telemetry(
 }
 
 fn init_tracer_provider(app_name: &str, collection_endpoint: &str) -> Tracer {
+    // Set a custom error handler to log OpenTelemetry errors with timestamps
+    global::set_error_handler(|error| {
```

Comment:
Mmmhh what is it doing ? Is it gonna log prepend every errors with this message?

Reply:
<img width="973" alt="Screenshot 2025-06-16 at 08 49 47" src="https://github.com/user-attachments/assets/8fbe4002-f81a-4675-b867-3aff707dd909" />

Reply:
And the before is on the issue linked